internal/pkg/deploy/cloudformation/project.go

```diff
@@ -63,6 +63,70 @@ func (cf CloudFormation) CreateProjectResources(project *archer.Project) error {
 	return nil
 }
 
+// GetProjectResourcesByRegion fetches all the regional resources for a particular region.
+func (cf CloudFormation) GetProjectResourcesByRegion(project *archer.Project, region string) (*archer.ProjectRegionalResources, error) {
+	resources, err := cf.getResourcesForStackInstances(project, &region)
+	if err != nil {
+		return nil, fmt.Errorf("describing project resources: %w", err)
+	}
+	if len(resources) == 0 {
+		return nil, fmt.Errorf("no regional resources for project %s in region %s found", project.Name, region)
+	}
+
+	return resources[0], nil
+}
+
+// GetRegionalProjectResources fetches all the regional resources for a particular project.
+func (cf CloudFormation) GetRegionalProjectResources(project *archer.Project) ([]*archer.ProjectRegionalResources, error) {
+	resources, err := cf.getResourcesForStackInstances(project, nil)
+	if err != nil {
+		return nil, fmt.Errorf("describing project resources: %w", err)
+	}
+	return resources, nil
+}
+
+func (cf CloudFormation) getResourcesForStackInstances(project *archer.Project, region *string) ([]*archer.ProjectRegionalResources, error) {
+	projectConfig := stack.NewProjectStackConfig(project, cf.box)
+	listStackInstancesInput := &cloudformation.ListStackInstancesInput{
+		StackSetName:         aws.String(projectConfig.StackSetName()),
+		StackInstanceAccount: aws.String(project.AccountID),
+	}
+
+	if region != nil {
+		listStackInstancesInput.StackInstanceRegion = region
+	}
+
+	stackInstances, err := cf.client.ListStackInstances(listStackInstancesInput)
```

Comment:
Can you help me understand what it would mean for the `ListStackInstancesInput` `StackInstanceRegion` value to be different from the `cf.client` configured region?

Reply:
Yea! So the stack set is in the same account as the project/codepipeline, etc (let's say us-west-2). 

If we add an env in another region, let's say us-east-1, then we spin that stack up, but we want to create an ECR repo in us-east-1. 

We update the stack-set in us-west-2 and tell it to add a new stack set instance in us-east-1. 

At this point, StackSets will create an actual stack in us-east-1, but we haven't had to make a cross region call yet, StackSets does it.

When we want to actually GET the output from that us-east-1 stack instance, we have to make a call configured for that region. So that's when we have to get a new regional client. (if we try to describe the us-east-1 stack is the project session - it'll throw a stack not found exception). 

Does that make sense?